client/my-sites/plan-features/header.jsx

```diff
@@ -58,7 +57,7 @@ class PlanFeaturesHeader extends Component {
 					newPlan && <Ribbon>{ translate( 'New' ) }</Ribbon>
 				}
 				{
-					current && <Ribbon>{ translate( 'Your Plan' ) }</Ribbon>
+					this.isPlanCurrent() && <Ribbon>{ translate( 'Your Plan' ) }</Ribbon>
```

Comment:
Side note (and outside of this PR's scope): you've probably noticed this is using the `getPlanClass` logic; that could probably use some improvement. What if we decide to use different `className` for monthly/yearly plans in the future?